lib/scsi_defs.h

```diff
@@ -97,5 +99,77 @@ namespace scsi_defs {
   } ABSL_ATTRIBUTE_PACKED;
   static_assert(sizeof(InquiryCommand) == kInquiryCommandSize);
 
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 60
+  enum class PeripheralQualifier: uint8_t {
+    kPeripheralDeviceConnected = 0b000,
+    kPeripheralDeviceNotConnected = 0b001,
+    kReserved = 0b010,
+    kNotSupported = 0b011
+  };
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 63
+  enum class TPGS : uint8_t {
+    kNotSupported = 0b00,
+    kImplicitAccess = 0b01,
+    kExcplicitAccess = 0b10,
+    kFullAccess = 0b11,
+  };
+
+  enum class ResponseDataFormat : uint8_t {
+    kObsolete0 = 0x0,
+    kObsolete1 = 0x1,
+    kCompliant = 0x2,
+  };
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 59
+  // Fields are arranged according to Big Endian format
+  struct InquiryData {
+    PeripheralQualifier peripheral_qualifier: 3;
+    PeripheralDeviceType peripheral_device_type: 5;
+    bool rmb: 1; // Removable Media Bit (RMB)
+    bool lu_cong: 1;
+    uint8_t reserved_1: 6;
+    Version version: 8;
+    bool reserved_2: 2;
```

Comment:
I think it would be more consistent to reserve bool to boolean fields only (i.e. one bit)

Reply:
That's a typo. Thanks for catching that.